### PR TITLE
Kyc button

### LINKED ISSUE #27: KYC

### PR DESCRIPTION
**Pull request by nelsontss**

### CODE REVIEW

File: web/src/App.js [hunk 1]
```diff
@@ -68,6 +69,13 @@ const App = (props) => {
                 component={LoanRequest}
                 wallet={wallet}
               />
+              <Route
+                auth={auth}
+                path="/kyc"
+                render={() => (
+                  <KycButton />
+                )}
+              />
```
Comment (Bishop19) [inline web/src/App.js]: Remember to remove this

File: web/src/containers/KycButton.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  const [open, setOpen] = useState(false)
+  const [verificationStatus, setVerificationStatus] = useState(props.verificationStatus)
+  const [intentId, setIntentId] = useState(props.intentId)
+  
+  useEffect(() => {
+    if(props.intentId !== undefined && !verificationStatus) {
+      isVerified(intentId)
+    }
+
+
```
Comment (Bishop19) [inline web/src/containers/KycButton.js]: run prettier

File: web/src/services/kyc.service.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+};
+
+const getVerificationResult = (verification_intent_id) => {
+  return instance
+    .get(`/external/get_verification_intent/${verification_intent_id}`)
+    .then(function(result) {
+      return result;
+    })
+    .catch((error) => {
+      // TODO : why is error overwritten in browser? we need to catch a specific error
```
Comment (Bishop19) [inline web/src/services/kyc.service.js]: aquele copy paste

**Review by Bishop19 — CHANGES_REQUESTED** [review]

**Review by Tiagod25 — APPROVED** [review]

File: web/src/services/investment.service.js [hunk 4]
```diff
@@ -16,12 +16,11 @@ instance.interceptors.request.use(
 );
 
 const newInvestment = (inv) => {
-  return instance.post("/new/",inv).then(function (response) {
+  return instance.post("/new/", inv).then(function (response) {
     console.log(response);
```
Comment (Bishop19) [inline web/src/services/investment.service.js]: return true on success and false on error?

**Review by Bishop19 — APPROVED** [review]

**Pull request merged**